official/utils/testing/scripts/presubmit.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+# Presubmit script that run tests and lint under local environment.
+# Make sure that tensorflow and pylint is installed.
+# usage: models >: ./official/utils/testing/scripts/presubmit.sh
+set +x
+
+# Assume the pwd is under models.
+cd official
+
+exit_code=0
+
+# Check lint:
+RC_FILE="utils/testing/pylint.rcfile"
+
+echo "===========Running lint test============"
```

Comment:
`git diff --name-only origin/master ./official/` has the nice property that you only have to care about stuff you touched. (And it makes linting faster.) The one caveat is it includes deleted files so you would have to do an existence test.

Reply:
If there are errors in things you didn't touch, the tests on Kokoro will fail as well. Printing them all will make us fix them faster. One hopes.

Reply:
Yea, currently running all the tests rather than the touched one. Since the file depend on each other, you cannot tell whether the test will break or not even its not touched.